docs/source/advanced/link-prediction.rst

```diff
@@ -83,6 +85,28 @@ GraphStorm provides three options to compute training losses:
 
     where ``G`` is the training graph.
 
+
+* **Adversarial Cross Entropy Loss**: The adversarial cross entropy loss turns a link prediction task into a binary classification task. We treat positive edges as 1 and negative edges as 0. In addition, adversarial cross-entropy loss adjusts the loss value of each negative sample based on its degree of difficulty. This is enabled by setting the ``--adversarial_temperature`` config.
+
+    The loss of positive edges is as:
+
+    .. math::
+        \begin{eqnarray}
+            loss = - y \cdot \log score + (1 - y) \cdot \log (1 - score)
+        \end{eqnarray}
+
+    where ``y`` is 1, ``score`` is the score value of the positive edges computed by the score function.
+
+    The loss of negative edges is as:
+
+    .. math::
+        \begin{eqnarray}
+            loss = - y \cdot \log score + (1 - y) \cdot \log (1 - score)
+            loss = softmax(score * adversarial_temperature) * loss
+        \end{eqnarray}
+
+    where ``y`` is 0, ``score`` is the score value of the negative edges computed by the score function and ``adversarial_temperature`` is a hyper-parameter.
```

Comment:
Add description about the final loss function, like ``loss = (avg(loss_pos) + avg(loss_neg)) / 2``.